samples/ImplicitFlow/OidcClientJs/public/env.js

```diff
@@ -0,0 +1,12 @@
+env = {
+    client: isDev() 
```

Comment:
Same remark as above, please remove the values that are specific to your deployment.